cdp_scrapers/instances/seattle.py

```diff
@@ -18,6 +22,43 @@
 
 ###############################################################################
 
+seat_image_uris: Dict[int, str] = {
+    1: "https://uploads.visitseattle.org/2015/03/"
+    "AlkiBeach_NickHallPhotography-720x313.jpg",
+    2: "https://uploads.visitseattle.org/2015/03/Safeco_DavidNewman-720x313.jpg",
+    3: "https://uploads.visitseattle.org/2015/03/"
+    "NoguchiNeedle_DavidNewman-2-720x313.jpg",
+    4: "https://uploads.visitseattle.org/2015/03/"
+    "Suzzallo_Reading_Room_NicholasBoos-720x313.jpg",
+    5: "http://www.seattle.gov/images/Departments/SDOT/BridgeStairsProgram/"
+    "AerialRendering_2020.jpg",
+    6: "https://uploads.visitseattle.org/2015/03/"
+    "BallardFarmersMarket_kallu-2-720x313.jpg",
+    7: "https://uploads.visitseattle.org/2015/03/QueenAnne_LempelZiv-720x313.jpg",
+    8: "https://www.seattle.gov/Images/Clerk/DistrictsMap.jpg",
+    9: "https://www.seattle.gov/Images/Clerk/DistrictsMap.jpg",
+}
```

Comment:
Why are these here / why doubly inject them? Aren't these the same URLs that are already in the static data JSON?

Not entirely happy with the district maps for 8 and 9 but can't think of anything better. I will ask the team in Slack though.

Reply:
Here is the long overdue answer to this simple question. This dictionary is used in `SeattleScraper.dump_static_info()` to generate `seattle-static.json`. When `seattle.py` is loaded, `known_persons` is read from `seattle-static.json` and passed into `LegistarScraper.__init__()`.
My thinking was, instead of manually hard-coding everything, call `dump_static_info()` once every N years to update `seattle-static.json`. I don't disagree it seems like overkill :sweat_smile: I am OK with just hard-coding these in `seattle-static.json` if you think that's better.

Reply:
ohhhh interesting. ya uhhhh i generally like to have only one place to edit changes soooo just the static file imo.

Reply:
> have only one place to edit changes

Absolutely. Years and years later I'm still forgetting my basics. Sold!

Reply:
I'm going to stop looking for the "right" picture for positions 8/9 a.k.a. at-large/citywide. Going with this:

![seattle](https://images.unsplash.com/photo-1523665307251-7f4e0e7b23fc?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8)

We can always change later. I just have to move on LOL.